### DISCUSSION TITLE
Transforming images in the table

### DISCUSSION THREAD

**Discussion by baharehesf** [pnp/powershell#1705] [Q&A]
Hi Everybody,

I'm recently working on the transformation project (SharePoint Online - Classic to Modern). I'm using PnP PowerShell. I have many pages that contains tables and in these tables, there are many images. But I saw that when I transfer theses tables, the URL of these specific images didn't transfer correctly. As the below image:

![git_TI](https://user-images.githubusercontent.com/97975542/158896450-5e1f5cf8-ad2c-401c-a804-bac657d99f5d.png)

The URL in classic: /sites/</****/>/en-ca/PublishingImages/Pages/<****>/<****>/Ammo1.png?ctag=220316
The URL in modern: /sites/****/PublishingImages/Pages/****/****/Ammo1.png

But at the end of the page, I can see all the images in the table that exist and show correctly.

Is there any solution for this?

@erwinvanhunen @KoenZomers @jansenbe @veronicageek

**Comment by ravindra8055** [comment]
I am also facing similar issues any luck fixing this issues?

**Reply by baharehesf** [reply]
Nope
